src/PrestaShopBundle/Form/Admin/Improve/International/Translations/ModifyTranslationsType.php

```diff
@@ -87,6 +89,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $noTheme = $this->trans('Core (no theme selected)', 'Admin.International.Feature');
 
+        $themeChoiceAttributes = [
+            $noTheme => [
+                'class' => 'js-no-theme',
+            ],
+        ];
+
+        if (isset($this->themeChoices[ThemeProviderDefinition::DEFAULT_THEME_NAME])) {
```

Comment:
If it is set, we override ?

Reply:
Yes, if it's set, we add a choice attribute to have a JS selector

Reply:
But in this case, you replace the array. You don't merge it.

Reply:
🤔 The $themeChoiceAttributes variable is declared just before. I don't replace the array, I add an item in it

Reply:
Oh yeah.